apps/src/templates/sectionAssessments/FeedbackDownload.jsx

```diff
@@ -16,7 +16,7 @@ import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
 const CSV_FEEDBACK_RUBRIC_HEADERS = [
   {label: i18n.studentName(), key: 'studentName'},
   {label: i18n.lessonNumber(), key: 'stageNum'},
-  {label: i18n.lessonName(), key: 'stageName'},
+  {label: i18n.lessonName(), key: 'lessonName'},
```

Comment:
this looks like it might conflict with #40893

Reply:
Done!